examples/gno.land/r/gnoland/home/home.gno

```diff
@@ -143,6 +144,15 @@ func socialLinks() ui.Element {
 	}
 }
 
+func playgroundSection() ui.Element {
+	return ui.Element{
+		ui.H3("Gno Playground, a DevX tool to write and deploy smart contracts directly on Gno.land"),
```

Comment:
It's very long.

What about:

```md
### Gno Playground

> a DevX tool to write and deploy smart contracts directly on Gno.land
```

Reply:
cc @ccomben I tend to agree with @thehowl

Reply:
The title would be easier to read and the page less heavy, so more pleasant to scroll.

Reply:
cc @michelleellen

Reply:
@salmad3 any suggestions to make this title shorter?

Reply:
Maybe just "Create, deploy, and share smart contracts."

Reply:
@alexiscolin can we update it as "Build, Deploy, and Share Smart Contracts with Playground" and resolve this comment?

Reply:
Which is (visually) better in your opinion?

### Build, Deploy, and Share Smart Contracts with Playground

---

or:

### Gno Playground

> Build, Deploy, and Share Smart Contracts

---

This is bikeshedding at this point, so I don't care whatever we do, but I'm personally of the opinion that a header should be to the point and not screw me around with productspeech. Especially since we don't have visual tools to put emphasis on "Playground" in the title.

Whichever is fine, though. So long as the line doesn't literally wrap around.